group_vars/testnet.yml

```diff
@@ -1,9 +1,5 @@
 ---
 # maintains versions and can overwrite all.yml
-sui:
+chainflip:
   validator:
```

Comment:
lets have node and engine separately, there are times when there is a new engine version but they dont ask us to update the node version